pages/trial-procedures.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div id="main-content" class="text-wrapper w-full">
+    <h1 class="text-center mb-6">
+      Witchcraft trial procedure and evidence in Scotland
+    </h1>
+    <div>
+      <div class="lg:float-right lg:w-1/2 lg:ml-6 w-3/4 mx-auto">
+        <img src="../public/images/trial.png" class="w-full" />
```

Comment:
I think we should probably add borders to the images like we did for an earlier PR just to be consistent. Cheers Ruby

Reply:
done, they aren't very visible though

Reply:
thank you. Aye, they aren't too obvious but I think it is good to have them. Thanks for adding these.